angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/marketplace-details.html

```diff
@@ -47,8 +47,9 @@
 			      <span class="fa fa-plus"></span> Add to home
 			    </button>
 
-		    <!-- Launch button with access -->
-		    <a class="btn btn-success" href="{{::portlet.maxUrl}}" target="{{::portlet.target}}" ng-if="portlet.canAdd"><span class="fa fa-arrow-circle-o-right"></span> Launch</a>
+		    <sub class="sr-only" id="goTo-portlet.title-{{::portlet.fname}}">go to this app</sub>
```

Comment:
How about `go to {{::portlet.title}}` rather than `go to this app`?  Seems like a worrying use of the term "app". :smile: